scripts/linters/test_files/invalid_trailing_comments.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Python file with invalid syntax, used by scripts/linters/
```

Comment:
Which specific test is using this file? I see all tests are creating a temporary file and then running the check?

Reply:
Ah thanks, I forgot to remove that file